### PR TITLE
remove the sentence with charter, add FAQ menu, add sidebar blocks, removed unnecessary files

### PR DESCRIPTION
**Pull request by flora8984461**
remove the sentence with charter, add FAQ menu, add sidebar blocks, removed unnecessary files

but the sidebar needs some more help,
- cannot get 2 separate blocks, one for "get help", another for "LTS Flyovers", 
- is there a simple way to have one sidebar showing in all pages? Now I am copying paste a lot in config file.

Signed-off-by: Yi Liu <yi.liu@eclipse-foundation.org>

### CODE REVIEW

**Comment by chrisguindon** [comment]
@flora8984461 The logic for the sidebar is located here:
https://github.com/EclipseFdn/hugo-solstice-theme/blob/master/layouts/partials/sidebar.html|

In order to replace or change this behavior, you can copy this file in the layout/partials folder of the lts.eclipse.org and make changes to it.

For example, in this file you can write more logic to include sidebar blocks on all page. Another option is to hardcode these new sidebar block HTML in that file.

**Comment by flora8984461** [comment]
> @flora8984461 The logic for the sidebar is located here:
> [https://github.com/EclipseFdn/hugo-solstice-theme/blob/master/layouts/partials/sidebar.html|](https://github.com/EclipseFdn/hugo-solstice-theme/blob/master/layouts/partials/sidebar.html%7C)
> 
> In order to replace or change this behavior, you can copy this file in the layout/partials folder of the lts.eclipse.org and make changes to it.
> 
> For example, in this file you can write more logic to include sidebar blocks on all page. Another option is to hardcode these new sidebar block HTML in that file.

After referring to this, find a workaround.

File: config.toml [hunk 1]
```diff
@@ -57,50 +57,78 @@ pluralizeListTitles = false
...
-  url = "/resources"
+  url = "/resources/"
   weight = 3
 
+[[menu.main]]
+  name = "FAQ"
+  url = "/about/faq/"
+  weight = 4
+
+#######################
```
Comment (autumnfound) [inline config.toml]: We shouldn't include a comment like this that isn't present in other sites. If we need to split the configs, we can look at doing that as it is [supported by Hugo](https://gohugo.io/getting-started/configuration/#configuration-directory), and we've done it in the Hugo Solstice Theme exampleSite configuration.

File: layouts/partials/sidebar.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    {{ .Scratch.Set "currentSection" $currentPage.Section }}
+  {{ end }}
+  
+  {{$currentSection := .Scratch.Get "currentSection" }}
+
+  {{ range .Site.Menus.sidebar }}
+    {{$parentItem := . }}
+    
+    <!-- as long as has children, removed condition equal currentSection, to show on all pages -->
+      {{ if and (.HasChildren) }}
```
Comment (autumnfound) [inline layouts/partials/sidebar.html]: this can just be `{{ if .HasChildren }}` as there is only one condition

**Review by autumnfound — CHANGES_REQUESTED** [review]

**Review by autumnfound — APPROVED** [review]

**Pull request merged**